docs/GETTING_STARTED_GUIDE.md

```diff
@@ -65,7 +65,7 @@ A Storage Administrator can execute the installer or rpm package as a root user
 ```
```

Comment:
No changes required here.  Content has moved to the common doc site so please make changes there instead

Reply:
removed